ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -16,6 +15,14 @@ readonlyrest:
       auth_key: kibana:kibana
       verbosity: error
 
+    - name: "Accept RO requests to NL indices(LDAP)"
+      auth_key: test:test
+      kibana:
+         access: ro
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 9

(indentation)